packages/plugin-koa/package.json

```diff
@@ -23,6 +23,6 @@
   },
   "devDependencies": {
     "@bugsnag/core": "^8.0.0",
-    "@types/koa": "^2.11.3"
+    "@types/koa": "2.13.4"
```

Comment:
is this pinned due to an issue with unit testing? would be good to add a task to update this and validate it